dbms/src/Storages/KVStore/FFI/JointThreadAllocInfo.cpp

```diff
@@ -0,0 +1,184 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Common/TiFlashMetrics.h>
+#include <Storages/KVStore/FFI/JointThreadAllocInfo.h>
+#include <Storages/KVStore/FFI/ProxyFFI.h>
+
+#include <mutex>
+#include <thread>
+#include <unordered_set>
+
+namespace DB
+{
+
+JointThreadInfoJeallocMap::JointThreadInfoJeallocMap()
+{
+    monitoring_thread = new std::thread([&]() {
```

Comment:
Setup the thread_name for the `monitoring_thread`

Reply:
fixed